integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkIT.java

```diff
@@ -278,6 +278,36 @@ void testAlterTableSetAndRemoveProperty() {
     Assertions.assertTrue(newProperties.containsKey("key2"));
   }
 
+  @Test
+  void testAlterTableAddAndDeleteColumn() {
+    String tableName = "test_column";
+    dropTableIfExists(tableName);
+
+    createSimpleTable(tableName);
+    checkTableColumns(tableName, getSimpleTableColumn(), getTableInfo(tableName));
+
+    sql(String.format("ALTER TABLE %S ADD COLUMNS (col1 string)", tableName));
+    List<SparkColumnInfo> addColumn =
```

Comment:
could you build new columns with `getSimpleTableColumn`,  because  `getSimpleTableColumn` is bind to `createSimpleTable`, if we change `createSimpleTable` we only need to change `getSimpleTableColumn`.

Reply:
done